config/opensearch_dashboards.yml

```diff
@@ -408,10 +408,10 @@
 
 # @experimental Set the enabled value to true and provide an AG-UI agent to enable chat
 # chat:
-  # enabled: true
-  # agUiUrl: "http://your-ai-agent-url:port"
-  # mlCommonsAgentId: "olly-chat-agent-id"
+# enabled: true
+# agUiUrl: "http://your-ai-agent-url:port"
+# mlCommonsAgentId: "olly-chat-agent-id"
```

Comment:
Should we keep the indents?

Reply:
@ruchidh can you just revert the change here?